src/_pytest/hookspec.py

```diff
@@ -523,6 +523,11 @@ def pytest_report_header(config, startdir):
     :param _pytest.config.Config config: pytest config object
     :param startdir: py.path object with the starting dir
 
+    .. note:
+
+        The returned (set of) lines is reversed, i.e. `trylast=True` has to
```

Comment:
Say Plugin1 returns Lines1 and Plugin2 returns Lines2. Which of the following describes what happens?

1. `"\n".join(reversed([Lines1, Lines2]))`
2. `"\n".join(reversed(Lines1 + Lines2))`
3. `"\n".join(reversed(Lines1) + reversed(Lines2))`

Reply:
`1.`

Ref: https://github.com/pytest-dev/pytest/blob/48c9f556ef17d67b820ac36944f5ee234dd624c6/src/_pytest/terminal.py#L640

Reply:
Got it. So I would rephrase slightly such that the first part of the sentence cannot be misconstrued. Suggestion:

Lines returned by a plugin are displayed *before* those of plugins which ran before it.
If you want to have your line(s) displayed first, use `trylast=True`.